client/src/components/JurisdictionAdmin/BatchRoundDataEntry/BatchDetails.tsx

```diff
@@ -0,0 +1,472 @@
+        // A special tab bar with a sheet name form input replacing the selected tab, rendered here
+        // instead of in the parent component so that we can create a separate react-hook-form form
+        // per sheet (sharing the form across sheets requires diligent resetting)
+        <Tabs id={batch.name} selectedTabId={selectedTabId}>
+          {tabs.map(tab =>
+            selectedTabId === tab.id ? (
+              <input
+                aria-label="Sheet Name"
+                className={classnames(
+                  Classes.INPUT,
+                  errors.name && Classes.INTENT_DANGER
+                )}
+                key={tab.id}
+                name="name"
+                readOnly={isSubmitting}
+                ref={register({
+                  required: true,
+                })}
+              />
+            ) : (
+              <Tab disabled id={tab.id} key={tab.id}>
+                {tab.name}
+              </Tab>
+            )
+          )}
+          <Tabs.Expander />
+          <Button icon="add" minimal disabled>
+            Add Sheet
+          </Button>
+        </Tabs>
+      ) : (
+        // When the above tab bar isn't rendered, still render a hidden input to house the sheet
+        // name. Rendering no input at all interferes with react-hook-form's state management,
+        // specifically its isDirty tracking
+        <input name="name" ref={register()} style={{ display: 'none' }} />
+      )}
+
+      <div
+        // Since we have to render two tab bars, we aren't able to use the tab bar's built-in
+        // `panel` prop and have to manually apply attributes for tab panel accessibility
+        aria-labelledby={`${Classes.TAB_PANEL}_${batch.name}_${selectedTabId}`}
+        role="tabpanel"
+      >
+        <BatchResultTallySheetTable>
+          <thead>
+            <tr>
+              <th>Choice</th>
+              <th>Votes</th>
+            </tr>
+          </thead>
+          <tbody>
+            {contest.choices.map(choice => (
+              <tr key={choice.id}>
+                <td>{choice.name}</td>
+                <td>
+                  {isTotalsSheet ? (
+                    <span>
+                      {sum(sheets.map(sheet => sheet.results[choice.id]))}
+                    </span>
+                  ) : (
+                    <>
+                      <input
+                        aria-label={`${choice.name} Votes`}
+                        className={classnames(
+                          Classes.INPUT,
+                          errors.results?.[choice.id] && Classes.INTENT_DANGER
+                        )}
+                        name={`results[${choice.id}]`}
+                        readOnly={isSubmitting}
+                        ref={register({
+                          min: 0,
+                          required: true,
+                          valueAsNumber: true,
+                        })}
+                        type="number"
```

Comment:
I feel like the increment/decrement buttons that show up on the right side of the input are probably not so useful here. They also aren't aligned correctly. Not a huge deal, but if they're easy to remove, that could be nice
<img width="222" alt="Screen Shot 2022-10-31 at 11 11 18 AM" src="https://user-images.githubusercontent.com/530106/199079454-0fbec05e-9bc0-4643-afb3-23af1bc7bcf3.png">